video/out/vo_gpu_next.c

```diff
@@ -1001,8 +1005,8 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
         pl_swapchain_colorspace_hint(p->sw, NULL);
     }
 
-    struct pl_swapchain_frame swframe;
     struct ra_swapchain *sw = p->ra_ctx->swapchain;
+    struct pl_swapchain_frame swframe;
```

Comment:
This change is not really needed.

Reply:
Oops some weird leftover that wasn't supposed be here